### ISSUE TITLE
Wx Python 3 Support

### ISSUE THREAD

**Issue by dougbrn** [UWMRO/TCC#26]
This has been on my mind lately so I figured I'd make an issue just to flag a potential down the road issue that we might run into with Bifrost. Currently, Bifrost and the TCC run in Python 2, mainly because Bifrost was written using the WxPython GUI library. Wx will never be directly ported to Python 3, and is instead attempting to be implemented through a fan project called Project Phoenix (https://wxpython.org/Phoenix/docs/html/MigrationGuide.html). With Python 2 being deprecated in 2020, this puts Bifrost+TCC in a situation regarding it's future viability. The simplest solution is to just freeze it in Python 2, where we just live with the software ecosystem that surrounds it now. This might not be ideal in that we won't be able to capitalize on any improvements made to the packages in astropy (or anything else) that we reference in the Bifrost + TCC. The other option is to port Bifrost + TCC to Python 3 using Wx Phoenix. From what I've heard, Phoenix has ported the vast majority of what was contained in Wx, so *potentially* it could be just plug and play with small syntax changes needed here and there. Something to think about.

**Comment by jwhueh** [comment]
Or rewrite in pyqt/pyside which has massive momentum and has excellent
support.

Pyqtgraph is my preferred these days.

**Comment by dougbrn** [comment]
I completely agree that a conversion to pyqt would be the ideal, it really seems to have pulled away from wx competition wise in the last several years. It would definitely be a larger undertaking though. Could be a good project for newer AUEG members to take on if the project load is ever too light!
